backend/api/Mapping/BaseMap.cs

```diff
@@ -11,6 +11,8 @@ public void Register(TypeAdapterConfig config)
             config.NewConfig<Entity.BaseEntity, Models.BaseModel>()
                 .Map(dest => dest.CreatedOn, src => src.CreatedOn)
                 .Map(dest => dest.UpdatedOn, src => src.UpdatedOn)
+                .Map(dest => dest.UpdatedByName, src => src.UpdatedBy != null ? src.UpdatedBy.DisplayName : src.CreatedBy != null ? src.CreatedBy.DisplayName : "unknown")
```

Comment:
Yeah, so I'm not entirely sure this is the best way forward, maybe we should be returning an actual user object here instead of these specific fields?